frontend/apps/reader/modules/readergesture.lua

```diff
@@ -359,6 +359,146 @@ function ReaderGesture:addToMainMenu(menu_items)
             },
         },
     }
+    menu_items.gesture_intervals = {
+        text = _("Gesture intervals"),
+        sub_item_table = {
+            {
+                text = _("Double tap interval"),
+                callback = function()
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local GestureDetector = require("device/gesturedetector")
+                    local items = SpinWidget:new{
+                        text = T(_([[
+Set double tap interval.
+Time interval is in miliseconds.
+Acceptable value from 100 (0,1 sec) to 2000 (2 sec).
```

Comment:
I don't like "acceptable value". I'll think about it more later. :-)

Reply:
`Accepts value from` if you want to keep the same verb. Otherwise, `supports`, `handles`?

Reply:
I'd make it something like this.

> Set double tap interval in milliseconds.
> 
> The interval value can range from 100 (0.1 seconds) to 2000 (2 seconds).

Reply:
Thanks :)
Changes have been made.

![obraz](https://user-images.githubusercontent.com/22982594/61773734-a9456180-adf5-11e9-9550-ef1d664538e3.png)